### ISSUE TITLE
Enderpearls appear as taking impact on transparent blocks

### ISSUE THREAD

**Issue by Leejit** [XFuryMCPE/EnderPearls#1]
Ball launches from chest, not hand, it drops too quickly, and doesn't work while standing in water...derp.

Make a tp bow plug ! But like, if u can... make it toggle, so you can do like /tpbow & it just toggles it on or off... so u can shoot an arrow and tp there !! THAT'D be LIT

**Comment by Leejit** [comment]
is there a way to just make em a bit more powerful when throwing... so they go like...double the distance ?

**Comment by XFuryMCPE** [comment]
What do you mean launches from hand..? And you can adjust the speed in the config.yml

**Comment by Leejit** [comment]
nvm bout where it tosses from

& what value in the config ? 

But, it has some little bugs. 

this is my favorite plugin !

Though, throwing in water, or while in a tall grass, or if the ball lands on glass or some other things, it doesn't seem to work. 

Eh, I can tell my players to stfu. this is tricky, but MY FAVE. 

but yah, where in the config ? 

& can you possibly make a toggle for bows to tp/shoot :) that's be OP a.f

**Comment by XFuryMCPE** [comment]
plugins/EnderPearls/config.yml

**Comment by Leejit** [comment]
Noob... go look AT UR OWN CONFIG where does it mention where to change the speed/power of the throw ? LOL.  but t.y for the help.

**Comment by Leejit** [comment]
![screenshot_20160807-125555](https://cloud.githubusercontent.com/assets/17371144/17463846/5b57340e-5c9e-11e6-9b83-edda0c7dae52.png)

**Comment by Leejit** [comment]
So strange... I guess my copy doesn't have w.e you think it does. LOL.

**Comment by XFuryMCPE** [comment]
Oh wait that was a different plugin I made that allowed adjusted speed. This doesn't support that at the moment.

**Comment by Leejit** [comment]
:P hehehehe. Welp, got that other plug still ? I'd tinker. 

Would be cool. 

Uhm... I heard from a lil birdy you use snowballs on ur server... I tested it on mine, and it tosses two balls.. one nice and far, the snowball, and another not near as far, the ball you teleport to.

**Comment by XFuryMCPE** [comment]
I will attempt to add support for snowballs.

**Comment by Leejit** [comment]
Nahh.... Not necessary.

Instead, make what you have go double the distance... or not fall as fast.

**Comment by XFuryMCPE** [comment]
No. I'm going to fix ALL of the bugs you have explained, thank you for reporting them.

**Comment by Leejit** [comment]
Your call  ! You've made two of my favorite plugins. 

Just comment here and I'll D.L and test the shit out of it. 

Also, Takeover aka MindControl doesnt mimic head movement... but still, fckn sick plug ! I tested it, smidge glitchy, but for what it does, hella impressed. 

Good work dude.
